truss/templates/server.Dockerfile.jinja

```diff
@@ -4,7 +4,7 @@ ARG PYVERSION={{config.python_version}}
     {% if config.resources.use_gpu %}
 FROM baseten/baseten-huggingface-transformer-server-gpu-base:latest
     {% else %}
-FROM baseten/baseten-huggingface-transformer-server-base:latest
+FROM huggingface/transformers-cpu:4.18.0
```

Comment:
This assumes we won't need to make additions to the image right. If we do, we'd change it back to our wrapper baseten image.

Reply:
If the changes are small then we can make them inline here. If they get complex then it would be better to use the wrapper. I think creating a wrapper for a couple of lines is not worth the hassle of having to push extra images. 
I don't like it that the GPU and non-gpu cases are inconsistent here. Let me leave a todo to fix this.

Reply:
Actually there is one advantage of using baseten wrapper image, we can update that without user having to update their library version. There are tradeoffs, I'm not sure which is better. But it would be best to be consistent whichever way we decide to go. I've left a todo, I'll try to take care of it soon.